cmd/macpassd/dbConnection.go

```diff
@@ -0,0 +1,89 @@
+package main
+
+import (
+	"database/sql"
+	"fmt"
+	"log"
+	"os"
+	"time"
+
+	_ "github.com/mattn/go-sqlite3"
+)
+
+type MacRegistration struct {
+	mac  string
+	id   int
+	user string
+}
```

Comment:
A questo punto direi che puoi togliere questa struttura e usare quella definita da me in main. Rendila globale mettendo la prima letter maiuscola e aggiungici pure un campo id